index.html

```diff
@@ -1,13 +1,26 @@
 <!DOCTYPE html>
-<html lang="en-us">
-
+<html lang="en">
```

Comment:
Change this back to "en-us" with next update.

Reply:
OK